main.go

```diff
@@ -1479,10 +1510,16 @@ func loadPkgCache(lpkg *listedPackage, pkg *types.Package, files []*ast.File, in
 		}
 		return loaded, nil
 	}
-	return computePkgCache(fsCache, lpkg, pkg, files, info, ssaPkg)
+
+	return pkgCache{}, fmt.Errorf("pkg: %s not cached yet", lpkg.ImportPath)
 }
 
-func computePkgCache(fsCache *cache.Cache, lpkg *listedPackage, pkg *types.Package, files []*ast.File, info *types.Info, ssaPkg *ssa.Package) (pkgCache, error) {
+func computePkgCache(lpkg *listedPackage, pkg *types.Package, files []*ast.File, info *types.Info, ssaPkg *ssa.Package) (pkgCache, error) {
+	fsCache, err := openCache()
+	if err != nil {
+		return pkgCache{}, err
+	}
```

Comment:
just like GARBLE_CACHE, this extra call to openCache should either be reverted or explained

Reply:
I'm not sure a doc string makes sense here. "loadPkgCache" is just no longer automatically computing any package passed to it.